cmd/dlv/cmds/commands.go

```diff
@@ -716,13 +734,27 @@ func traceCmd(cmd *cobra.Command, args []string, conf *config.Config) int {
 					continue
 				}
 				for i := range addrs {
-					_, err = client.CreateBreakpoint(&api.Breakpoint{
-						Addr:        addrs[i],
-						TraceReturn: true,
-						Stacktrace:  traceStackDepth,
-						Line:        -1,
-						LoadArgs:    &terminal.ShortLoadConfig,
-					})
+					if traceFollowCalls > 0 {
+						_, err = client.CreateBreakpoint(&api.Breakpoint{
+							Addr:             addrs[i],
+							TraceReturn:      true,
+							Stacktrace:       20,
+							Line:             -1,
+							LoadArgs:         &terminal.ShortLoadConfig,
+							TraceFollowCalls: traceFollowCalls,
+							RootFuncName:     regexp,
+						})
+					} else {
+						_, err = client.CreateBreakpoint(&api.Breakpoint{
+							Addr:             addrs[i],
+							TraceReturn:      true,
+							Stacktrace:       traceStackDepth,
+							Line:             -1,
+							LoadArgs:         &terminal.ShortLoadConfig,
+							TraceFollowCalls: traceFollowCalls,
+							RootFuncName:     regexp,
+						})
+					}
```

Comment:
Same as previous comment, change the value of `traceStackDepth` instead of duplicating the call.